packages/quiz/src/utils/submit-answer.ts

```diff
@@ -9,10 +9,20 @@ export default async function handleSubmitAnswer(context: QuizContext) {
     questionSet,
     answer,
   } = context
+
+  // TODO check Sanity API structure versus static JSON structure
+  //      they should be the same! It appears as though that might not be the case
+  let id =
+    findKey(questionSet, currentQuestion) ||
+    findKey(
+      questionSet,
+      (question) => question.question === currentQuestion.question,
+    )
```

Comment:
the id was parsing as `undefined` so I was curious if there was a disparity between Sanity and JSON data formats. If so, we should align them.

Reply:
https://github.com/skillrecordings/products/blob/876fc0ac6a52e0c77571d161dec38bd033a137a4/apps/cssdestructured/src/pages/answer.tsx#L28-L50

Reply:
here's where it gets transformed:

https://github.com/skillrecordings/products/blob/acf8ceae4f2e969794fc83e21875ef8bce5a5ff3/packages/quiz/src/utils/transform-sanity-quiz.ts#L10-L29